src/Plugin/Field/FieldFormatter/StrawberryCitationFormatter.php

```diff
@@ -10,6 +10,7 @@
 use Drupal\Core\Session\AccountInterface;
 use Drupal\Core\Template\TwigEnvironment;
 use Drupal\format_strawberryfield\EmbargoResolverInterface;
+use Drupal\pathauto\MessengerInterface;
```

Comment:
Hi, this interface is not the right one for the Messenger. Check other plugins or inspect $this->messenger()